src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -27,23 +27,53 @@ import { Title } from 'lib/ui/typography/Title.js'
 
 export interface AccessCodeTableProps {
   deviceId: string
+  accessCodeFilter?: (
+    accessCode: AccessCode,
+    searchInputValue: string
+  ) => boolean
+  accessCodeComparator?: (
+    accessCodeA: AccessCode,
+    accessCodeB: AccessCode
+  ) => number
   onBack?: () => void
   className?: string
 }
 
+type AccessCode = UseAccessCodesData[number]
+
+const defaultAccessCodeFilter = (
+  accessCode: AccessCode,
+  searchInputValue: string
+) => {
+  if (searchInputValue === '') return true
+  return new RegExp(searchInputValue, 'i').test(accessCode.name ?? '')
+}
+
 export function AccessCodeTable({
   deviceId,
   onBack,
+  accessCodeFilter = defaultAccessCodeFilter,
```

Comment:
Nice, good idea! Out of curiosity, was this a requested feature?

Reply:
Yup :)